lib/version/version.go

```diff
@@ -1,4 +1,4 @@
 package version
 
 // Pre-built binaries will have version set during build time.
-var Version = "????"
+var Version = "HEAD"
```

Comment:
+1 for `v0.0.14-prerelease` or `v0.0.13-HEAD` with this being updated by `release.sh`